wiki/spectrum/jamming/drone-jamming.md

```diff
@@ -1,9 +1,75 @@
-## Jamming
+##  Jamming Drones
+Jamming of drones comes in 2 different forms.
+1. using the [Spectrum Device](https://community.bistudio.com/wiki/Arma_3:_Spectrum_Device) - against individual drones
+2. using an **area denial jammer** - against all drones in a specific radius
+
+### Spectrum Device
+
 UGV drones can be jammed with a jammer antenna. By default, all UGV vehicles will broadcast a signal between 434MHz and 440MHz. By activating the antenna on the signal with left-mouse button when there is enough signal strength the drones AI will be disabled. The signal has to be higher than -40 for jamming to be possible. As long as you keep having the jammer activated the drone will just stand still where it is and do nothing. So if you jam it and move past it, and stop the jam, the drone will continue with what it was doing before. However, if the drone already spotted you and went into combat, when you stop jamming it will still be in combat. 
 
 ```admonish info
 When a drone is firing the machine gun it happens in bursts. When you apply the jamming and the AI is halted, it will still finish the round of burst fire before stopping. 
 ```
 
 Video:
-<iframe width="560" height="315" src="https://www.youtube.com/embed/VywVOCDEh9A?si=5_Vwq7i802DKGu72" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
\ No newline at end of file
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VywVOCDEh9A?si=5_Vwq7i802DKGu72" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+
+### Area denial jammer
+Player controlled and autonomous drones can not operate within the effective range of the area denial jammer.
+- Autonomous drones will lose their autonomous capabilities (in technical terms [disableAI "all"](https://community.bistudio.com/wiki/disableAI) is called)
+  - and if being jammed for long enough will attempt to land.
+- Any player that is currently controlling either the Driver or Gunner position of the drone is getting disconnected from it.
+
+The screenshot below shows 
+- the default 3D model of the area denial jammer on the left
+- and the message a drone pilot will see when getting disconnected on the upper right.
+
+![image](https://cdn.discordapp.com/attachments/402412803153199104/1202531922098786335/image.png?ex=65cdcc20&is=65bb5720&hm=ac8674aebedf7e16bd3f35fe5b06f5a78eb992687304a4c05b87abc1e2c2013b&)
```

Comment:
@b-mayr-1984 You happen to have this image in raw format? You can just send it to me on discord and I will upload it. This link is dead as "its no longer available on discord".

Reply:
done
(and also here)
![drone is jammed](https://github.com/Crowdedlight/Crows-Electronic-Warfare/assets/76476468/efe4b218-b76e-4ab1-ad3f-4ff5bc7f45a3)